lms/static/js/Markdown.Editor.js

```diff
@@ -1130,14 +1130,31 @@
             cancelButton.onclick = function() { return close(true); };
 
             if (imageUploadHandler) {
+                okButton.disabled = true;
                 var startUploadHandler = function() {
-                    document.getElementById('file-upload').onchange = function() {
+                    document.getElementById('file-upload').onchange = function(event) {
                         imageUploadHandler(this, urlInput);
                         urlInput.focus();
 
+                        var lengthOfDescriptionField = $.trim(descInput.value).length;

```

Comment:
always try to declare variables at top of method or file

Reply:
Done

Reply:
Done